azurerm/internal/services/frontdoor/tests/resource_arm_front_door_firewall_policy_test.go

```diff
@@ -272,7 +272,7 @@ resource "azurerm_frontdoor_firewall_policy" "test" {
     }
   }
 
-  %s
+  %[3]s
```

Comment:
This test must have been failing before this PR, this was passing in the region text before, in my case "West Europe" was randomly being placed in the resource body